huxflux/huxflux.js

```diff
@@ -1,10 +1,30 @@
 /**
- * @returns {Number[]} Lista med sju nummer
+ * @returns {Number[]}
  */
 function huxflux() {
-    return; // Returnera din lista med nummer här
+    let Number = [];
+
+    let recursion = () => {
+        let newNum = Math.floor(Math.random() * 36);
+        if (Number.indexOf(newNum) == -1 && newNum !== 0) {
+            return newNum;
+        } else {
+            return recursion();
+        }
+    }
+
+    for (let i = 0; i < 7; i++) {
+            Number.push(recursion());
+    }
+    return Number;
+}
```

Comment:
Awesome job with the randomising function! Few things that are good practice and you should consider: 
 - Since it's an arrow function, as a good practice should  be "const" and not "let" since you might not want to re assign the whole function 
 - Comparison and logical operator in line 9, better practice to use '===" rather than '==', the first one compares the value and type of data, the same doesn't happen on the one you used :)